frontend/src/components/messages/BasicMessages.vue

```diff
@@ -0,0 +1,178 @@
+<!--
+ Copyright (c) 2020 - for information on the respective copyright owner
+ see the NOTICE file and/or the repository at
+ https://github.com/hyperledger-labs/organizational-agent
+
+ SPDX-License-Identifier: Apache-2.0
+-->
+<template>
+  <chat-window
+      :current-user-id="currentUserId"
+      :responsive-breakpoint="responsiveBreakpoint"
+      :rooms="rooms"
+      :rooms-loaded="roomsLoaded"
+      :room-id="roomId"
+      :messages="messages"
+      :messages-loaded="messagesLoaded"
+      :message-actions="messageActions"
+      @fetch-messages="fetchMessages"
+      @send-message="sendMessage"
+      :show-add-room="false"
+      :show-audio="false"
+      :show-files="false"
+      :show-emojis="false"
+      :show-reaction-emojis="false"
+      :show-new-messages-divider="false"
+      :text-formatting="false"
+      :text-messages="textMessages"
+  />
+</template>
+
+<script>
+  import ChatWindow from 'vue-advanced-chat'
+  import 'vue-advanced-chat/dist/vue-advanced-chat.css'
+  import * as partnerUtils from "@/utils/partnerUtils";
+  import partnerService from "@/services/partnerService";
+  import { mapMutations } from 'vuex';
+  import {CHAT_CURRENT_USERID} from "@/constants";
+  import {formatDateLong} from "@/filters";
+
+  export default {
+    components: {
+      ChatWindow
+    },
+    created() {
+    },
+    data() {
+      return {
+        currentUserId: CHAT_CURRENT_USERID,
+        responsiveBreakpoint: 9999,
+        rooms: [],
+        roomsLoaded: true,
+        loadingRooms: false,
+        roomId: null,
+        messages: [],
+        messagesLoaded: true,
```

Comment:
This is toggled in this.fetchMessages(...) but never evaluated?

Reply:
this is used by the `chat-window` component, it toggles the visual spinner